libc/startup/linux/CMakeLists.txt

```diff
@@ -1,19 +1,53 @@
+# This function merges multiple objects into a single relocatable object
```

Comment:
Should I add a link to the relocatable linkage post? https://maskray.me/blog/2022-11-21-relocatable-linking